src/MongoDB/Query.c

```diff
@@ -90,6 +90,15 @@ static bool php_phongo_query_opts_append_document(bson_t* opts, const char* opts
 		}                                                                                                            \
 	}
 
+#define PHONGO_QUERY_OPT_BOOL_DEPRECATED(opt, zarr, key)                                                                    \
+	if ((zarr) && php_array_existsc((zarr), (key))) {                                                                       \
+		php_error_docref(NULL TSRMLS_CC, E_DEPRECATED, "The '%s' option is deprecated and will be removed in a future release", key); \
```

Comment:
Nit: `\"%s\"` for consistency with the "Error appending" message (I did so in my suggestion above)

Reply:
I copied this from the PHPLIB change - thought it would be better to have it consistent!

Reply:
But I now see PHPLIB uses " too, so fixed.